java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -1372,4 +1372,270 @@ CompletableFuture<GlideString[]> xclaimJustId(
+     * // the response contains data in the following format:
+     * Map<String, Object> data = Map.of(
+     *     "length", 4L,
+     *     "radix-tree-keys", 1L,
+     *     "radix-tree-nodes", 2L,
+     *     "last-generated-id", "1719877599564-0",
+     *     "max-deleted-entry-id", "0-0",
+     *     "entries-added", 4L,
+     *     "recorded-first-entry-id", "1719710679916-0",
+     *     "entries", new Object {
+     *         "1719710679916-0",
+     *         new String[] {
+     *             "foo", "bar",
+     *             "foo", "bar2",
+     *             "some_field", "some_value"
+     *         },
+     *         "1719710688676-0",
+     *         new String[] {
+     *             { "foo", "bar2" },
+     *         },
+     *     },
+     *     "groups", new Map[] {
+     *         Map.of(
+     *             "name", "mygroup",
+     *             "last-delivered-id", "1719710688676-0",
+     *             "entries-read", 2L,
+     *             "lag", 0L,
+     *             "pel-count", 2L,
+     *             "pending", new Object[][] { {
+     *                     "1719710679916-0",
+     *                     "Alice",
+     *                     1719710707260L,
+     *                     1L,
+     *                 }, {
+     *                     "1719710688676-0",
+     *                     "Alice",
+     *                     1719710718373L,
+     *                     1L
+     *                 } },
+     *             "consumers", new Map[] {
+     *                 Map.of(
+     *                     "name", "Alice",
+     *                     "seen-time", 1719710718373L,
+     *                     "active-time", 1719710718373L,
+     *                     "pel-count", 2L,
+     *                     "pending", new Object[][] { {
+     *                             "1719710679916-0",
+     *                             1719710707260L,
+     *                             1L,
+     *                         }, {
+     *                             "1719710688676-0",
+     *                             1719710718373L,
+     *                             1L
+     *                         } }
+     *                 )
+     *             })
+     * }); // Detailed stream information for "my_stream".
+     * }</pre>
+     */
+    CompletableFuture<Map<String, Object>> xinfoStreamFull(String key);
+
+    /**
+     * Returns verbose information about the stream stored at key <code>key</code>.<br>
+     * The output is limited by first <code>10</code> PEL entries.
+     *
+     * @since Redis 6.0 and above.
+     * @see <a href="https://valkey.io/commands/xinfo-stream/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @return A <code>Map</code> of detailed stream information for the given <code>key</code>. See
+     *     the example for a sample response.
+     * @example
+     *     <pre>{@code
+     * // example of using the API:
```

Comment:
I guess you can just refer here to `xinfoStreamFull(String)` and tell that response has the same format.